orca-clouddriver/orca-clouddriver.gradle

```diff
@@ -15,6 +15,7 @@
  */
 
 apply from: "$rootDir/gradle/groovy.gradle"
+apply from: "$rootDir/gradle/kotlin.gradle"
```

Comment:
You can't mix Kotlin into a module that has Groovy, which is why the kotlin bits don't work - this is why I've been doing refactors Groovy -> Java, instead of Groovy -> Kotlin.